api/swagger.yaml

```diff
@@ -2102,6 +2102,16 @@ definitions:
                     SpreadDescriptor:
                       description: "label descriptor, such as engine.labels.az"
                       type: "string"
+          Platforms:
+            description: "An array of supported platforms."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                Architecture:
```

Comment:
> The OCI platform struct seems identical to manifestlist.PlatformSpec, which is what this is meant to consume.

It is slightly different. You should err on using the OCI version, where possible.

The only thing to keep in line here is that `Architecture` should always use a GOARCH value.